src/nester/nester_log.py

```diff
@@ -0,0 +1,115 @@
+        return record
+
+
+logger = logging.getLogger("nester")
+logger.setLevel(logging.INFO)
+
+log_file_path = Path.home() / "nester.log"
+file_handler = logging.FileHandler(log_file_path)
+
+formatter = ProjectLogFormatter(
+    "%(asctime)s - %(name)s - %(levelname)s - Project: %(projectname)s - Language: %(programming_language)s - Location: %(location)s"
+)
+
+file_handler.setFormatter(formatter)
+logger.addHandler(file_handler)
+
+
+def check_log_for_duplicate(projectname):
+    """
+    Check the nester.log file in the home directory, whether a given projectname has been taken already.
+    If no log file exists: Continue with the program.
+
+    :param projectname: Name of the project to be checked
+    :return: True/False depending on whether an entry has been found.
+    :rtype: Bool
+    """
+    try:
+        with log_file_path.open("r") as log_file:
+            log = log_file.readlines()
+    except FileNotFoundError:
+        print(f"\033[34mLog file not found. Moving on ...\033[0m")
+        return False
+
+    for line in log:
+        if projectname in line:
+            return True
+
+    return False
+
+
+def remove_log_entry(projectname):
+    """
+    Check if the given project appears in the log. If it does, remove the entry from the log.
+
+    :param projectname: The name of the project to be removed.
+    :return: None
+    """
+    try:
+        with log_file_path.open("r") as log_file:
+            log = log_file.readlines()
+    except FileNotFoundError:
+        print("\033[34mNo log file found! Moving on ...\033[0m")
+        return
+
+    entry_found = False
+    updated_lines = []
+
+    for line in log:
+        if projectname.split("/")[0] in line:
+            entry_found = True
+        else:
+            updated_lines.append(line)
+
+    if entry_found:
+        try:
+            with log_file_path.open("w") as log_file:
+                log_file.writelines(updated_lines)
+            print(f"\033[32mLog entry for '{projectname}' removed successfully.\033[0m")
+        except Exception as e:
+            print(f"\033[31mError while removing entry: {e}\033[0m")
+    else:
+        print(f"\033[34mNo log entry found for '{projectname}'\033[0m")
+
+
+def print_log_to_table():
+    try:
+        with open(log_file_path, "r") as log_file:
+            # Print table header
+            print(f"{'Project Name':<20} | {'Language':<10} | {'Location'}")
+            print("-" * 80)
+
+            for line in log_file:
+                project_name = re.search(r"[-\s]+Project:\s+(.*?)\s+-", line).group(1)
+                language = re.search(r"[-\s]+Language:\s+(.*?)\s+-", line).group(1)
+                location = re.search(r"[-\s]+Location:\s+(.*)", line).group(1)
+
+                print(f"{project_name:<20} | {language:<10} | {location}")
+            print()
+    except FileNotFoundError:
+        print("\033[31FATAL ERROR -- LOG FILE NOT FOUND - ABORTING!\033[0m")
```

Comment:
Nitpick: We should use the same error message for all FileNotFound errors. I think the other one in remove_log_entry fits better